src/lib/countryList.ts

```diff
@@ -0,0 +1,261 @@
+const countryList = [
+    "United Kingdom",
```

Comment:
Are we sure there is no Gov uk's component which can be used here? I am a bit skeptical of saving this list with us.

Reply:
I checked other projects where they using country/nationality, all of them are storing the list at project level only. Not sure if there is gov uk component.

Reply:
Yeah all the other repos I've looked at have stored the list of countries locally